loadable_apps/loadable_sample/wifiapp/binary_update.c

```diff
@@ -143,7 +143,7 @@ static int binary_update_download_binary(binary_update_info_t *binary_info, int
 		}
 		crc_hash = crc32part(buffer, read_size, crc_hash);
 		copy_size += read_size;
-		printf("Copy %s binary, %s [%d%%]\r", binary_info->name, filepath, copy_size * 100 / total_size);
+		printf("Copy %s binary to %s [%d%%]\r", binary_info->name, download_path, copy_size * 100 / total_size);
```

Comment:
line 118~127, now file system doesn't guarantees header is valid or not, so most of value in header should be used to calculate crc32...